context/app/static/js/components/workspaces/WorkspaceField/WorkspaceField.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useController } from 'react-hook-form';
+import TextField from '@mui/material/TextField';
+import IconButton from '@mui/material/IconButton';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import InputAdornment from '@mui/material/InputAdornment';
+
+import { useSnackbarStore } from 'js/shared-styles/snackbars';
+
+function WorkspaceField({ control, name, label, errors, value }) {
+  const { openSnackbar } = useSnackbarStore();
+  const { field } = useController({
+    name,
+    label,
+    control,
+    rules: { required: true },
+    defaultValue: value,
+  });
+
+  const handleCopyClick = () => {
+    navigator.clipboard.writeText(field.value).then(() => {
+      openSnackbar('Copied to clipboard.');
+    });
+  };
+
+  return (
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <TextField
+        InputLabelProps={{ shrink: true }}
+        label={label || name}
+        placeholder="Like “Spleen-Related Data” or “ATAC-seq Visualizations”"
```

Comment:
Is this placeholder relevant in all cases, or should this also be passed down as a prop?